src/openmc_nek_driver.cpp

```diff
@@ -194,7 +194,7 @@ void OpenmcNekDriver::init_temperatures()
         const auto& global_elems = cell_to_elems_.at(i);
         const auto& c = openmc_driver_->cells_[i];
 
-        for (int elem : global_elems) {
+        for (const auto &elem : global_elems) {
```

Comment:
Using auto (copy by value) is sufficient
```suggestion
        for (auto elem : global_elems) {
```